BREEZE.rst

```diff
@@ -137,131 +131,102 @@ You can always stop it via:
 
 .. code-block:: bash
 
    run-tests tests.core:TestCore.test_check_operators -- -s --logging-level=DEBUG
 
-The tests will run ``airflow db reset`` and ``airflow db init`` the first time you
-run tests in running container, so you can count on database being initialized.
+The tests run ``airflow db reset`` and ``airflow db init`` the first time you
+launch them in a running container, so you can count on the database being initialized.
 
 All subsequent test executions within the same container will run without database
-initialisation.
+initialization.
 
-You can also optionally add --with-db-init flag if you want to re-initialize
+You can also optionally add the ``--with-db-init`` flag if you want to re-initialize
 the database.
 
 .. code-block:: bash
 
    run-tests --with-db-init tests.core:TestCore.test_check_operators -- -s --logging-level=DEBUG
 
-Adding/modifying dependencies
+Adding/Modifying Dependencies
 -----------------------------
 
-If you change apt dependencies in the ``Dockerfile`` or add python pacakges in ``setup.py` or
-javascript dependencies in ``package.json``. You can add dependencies temporarily for one Breeze
-session or permanently in ``setup.py``, ``Dockerfile``, ``package.json``.
+If you need to change apt dependencies in the ``Dockerfile``, add Python pacakges in ``setup.py`` or
+add javascript dependencies in ``package.json``, you can either add dependencies temporarily for a single Breeze
+session or permanently in ``setup.py``, ``Dockerfile``, or ``package.json`` files.
 
-Installing dependencies for one Breeze session
+Installing Dependencies for a Single Breeze Session
 ..............................................
 
-You can install dependencies inside the container using 'sudo apt install', 'pip install' or 'npm install'
-(in airflow/www folder) respectively. This is useful if you want to test something quickly while in the
-container. However, those changes are not persistent - they will disappear once you
-exit the container (except npm dependencies in case your sources are mounted to the container). Therefore
-if you want to persist a new dependency you have to follow with the second option.
+You can install dependencies inside the container using ``sudo apt install``, ``pip install`` or ``npm install``
+(in ``airflow/www`` folder) respectively. This is useful if you want to test something quickly while you are in the
+container. However, these changes are not retained: they disappear once you
+exit the container (except for theh npm dependencies if your sources are mounted to the container). Therefore,
+if you want to retain a new dependency, follow the second option described below.
 
-Adding dependencies permanently
+Adding Dependencies Permanently
 ...............................
 
-You can add the dependencies to the Dockerfile, setup.py or package.json and rebuild the image. This
+You can add dependencies to the ``Dockerfile``, ``setup.py`` or ``package.json`` and rebuild the image. This
```

Comment:
This should happen automatically - what is "this"?

Reply:
Rebuilding the image should happen automatically when you re-enter breeze after changing any of the files.